languages/csharp/CommandRunner.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Bitwarden.Sdk;
+using System.Text.Json;
+
+internal class CommandRunner
+{
+
+    private readonly BitwardenSafeHandle _handle;
+
+    internal CommandRunner(BitwardenSafeHandle handle)
+    {
+        _handle = handle;
+    }
+
+    internal T? RunCommand<T>(Command command)
+    {
+        var req = command.ToJson();
```

Comment:
Consider using `JsonSerializer.Serialize` for this and `ClientSettings` since that would let us change the quicktype config to exclude the `FromJson` and `ToJson` helpers.

```suggestion
        var req = JsonSerializer.Serialize(command, Converter.Settings);
```

Reply:
Changed